reactions.json

```diff
@@ -97,24 +97,37 @@
     "Na": "sodium methoxide",
     "SOCl2": "chloromethane",
     "PCl5": "chloromethane",
-    "ZnCl2/ HCl": "chloromethane"
+    "PCl3": "chloroethane",
+    "ZnCl2 / HCl": "chloromethane",
+    "HCOOH": "methyl methanoate",
+    "Cu / 573K": "methanal",

```

Comment:
Temperature, Pressure conditions should be written inside () while different reagenets should be separated with /